### PR TITLE
make watch-only wallets visible on wallets page

### PR DESCRIPTION
**Pull request by itswisdomagain**
Watch-only wallets do not currently show up in the wallets page. This fixes that.

### CODE REVIEW

File: ui/decredmaterial/button.go [hunk 1]
```diff
@@ -195,6 +202,8 @@ func (bl ButtonLayout) Layout(gtx layout.Context, w layout.Widget) layout.Dimens
 }
 
 // TODO: Test to ensure this works!
+// TODO: Doesn't workk, if ib.colorStyle was nil before this method is called,
+// it is temporarily changed but when ib.Layout is called, it returns to nil.
 func (ib IconButton) ChangeColorStyle(colorStyle *values.ColorStyle) {
```
Comment (itswisdomagain) [inline ui/decredmaterial/button.go]: We'll need an issue to investigate other uses of this method which likely won't be working as expected.

**Comment by itswisdomagain** [comment]
Can test with this pub key `tpubVokqVVuZgvPWNuVQXUPMt3znUFJMqArBtuy4m8Xw2GGqBdXCWRqvPua21adM99n43CTn3dMnXnVVrncqkeCs4GukzgdxPRaFF2L322vdog6`.

File: ui/decredmaterial/button.go [hunk 2]
```diff
@@ -195,6 +202,8 @@ func (bl ButtonLayout) Layout(gtx layout.Context, w layout.Widget) layout.Dimens
 }
 
 // TODO: Test to ensure this works!
+// TODO: Doesn't work, if ib.colorStyle was nil before this method is called,
+// it is temporarily changed but when ib.Layout is called, it returns to nil.
```
Comment (Sirmorrison) [inline ui/decredmaterial/button.go:206]: When will these `TODO`s be resolved? is this something that can go in to this PR?

**Review by Sirmorrison — CHANGES_REQUESTED** [review]
The PR looks good... Just a few changes.
The option pop up width is too much to the right.
image

it should also have same alignment with the wallet pop up.
**wallet**
image

**watch only**
image

**Comment by Sirmorrison** [comment]
Please add a description to this PR.

File: ui/decredmaterial/button.go (hunk 2 above)
Comment (itswisdomagain) [reply ui/decredmaterial/button.go:206]: Will need to go into a separate issue to be examined separately.

**Review by Sirmorrison — APPROVED** [review]
LGTM

**Pull request merged**